src/System.Reflection.Metadata/tests/Metadata/MetadataReaderTests.cs

```diff
@@ -2122,12 +2114,24 @@ public void ValidateCustomAttribute()
             }
 
             Assert.Equal(0x37, i);
+        }
 
-            // ====================================================
-            reader = GetMetadataReader(NetModule.ModuleVB01, true);
+        [Fact]
+        public void ValidateModuleCustomAttributes()
+        {
+            // ModuleVb01
+            var modTypes = new int[] { 0x0a000003 };
+            var modParents = new int[] { 0x02000002 };
+            var modValues = new byte[][]
+            {
+                new byte[] { 0x01, 00, 0x10, 0x4d, 0x6f, 0x64, 0x56, 0x42, 0x44, 0x65, 0x66, 0x61, 0x75, 0x6c, 0x74, 0x50, 0x72, 0x6f, 0x70, 00, 00 }
+            };
+            
+            var reader = GetMetadataReader(NetModule.ModuleVB01, true);
+            var moduleDefinition = reader.GetModuleDefinition();
 
-            i = 0;
-            foreach (var caHandle in reader.CustomAttributes)
+            int i = 0;
+            foreach (var caHandle in moduleDefinition.GetCustomAttributes())
```

Comment:
Where possible, I tend to agree, but in this case we have a collection that isn't indexable IIRC.